Makefile.defs

```diff
@@ -73,6 +73,17 @@ else ifeq ($(TARGET), appimage)
 	export EMULATE_READER=1
 	export SDL=1
 	export APPIMAGE=1
+else ifeq ($(TARGET), debian)
+        export SDL=1
+        export EMULATE_READER=1
```

Comment:
You'll also need to adapt #644.

Reply:
Fair point. Not sure how to proceed,  since the debian target should be as portable as the emulator.

Maybe two more targets for amd64 and i686 to be able to build both from a multilib x64 debian and keep "debian" target as native?

I will upload the frontend part of this PR to show you my (possibly borked) logic :)

Reply:
@Frenzie done!